src/core.rs

```diff
@@ -271,27 +304,27 @@ impl<'a> Word<'a> {
                     continue;
                 }
 
-                if width > 0 && width + ch_width(ch) > line_width {
+                if width > 0 && width + ch_width(ch, self.tab_width) > line_width {
```

Comment:
Similarly here, we're stepping through the characters in `self.word`, so we should not expect to find any `'\t'` characters, right?

Reply:
Right, we shouldn't - but I think that, unless we want to add an additional, like, `ch_width_without_tabs` method to calculate the width of a character while assuming it's not a tab, there's no reason to not pass in `self.tab_width`, correct?

Reply:
Yeah, I think you're correct. It looks like it won't be necessary to know the tab width when constructing a `Word`.